.github/workflows/ci.yml

```diff
@@ -44,3 +44,16 @@ jobs:
         run: |
           source .venv/bin/activate
           pytest tests/
```

Comment:
Maybe there should be a new line between the end of the first job and the beginning of the second job?